src/components/filter-btn-container/index.jsx

```diff
@@ -1,12 +1,29 @@
 import React from "react";
 import Button from "components/base/button";
+import { useDispatch } from "react-redux";
+import { filterTask } from "store/actions/filter";
+import {
+  TASK_FILTER_ALL,
+  TASK_FILTER_COMPLETED,
+  TASK_FILTER_INCOMPLETED,
+} from "utils/const";
 
 function FilterBtnContainer() {
+  const dispatch = useDispatch();
+
   return (
     <div>
-      <Button>All</Button>
-      <Button>Incomplete</Button>
-      <Button>Complete</Button>
+      <Button onButtonClick={() => dispatch(filterTask(TASK_FILTER_ALL))}>
+        All
+      </Button>
+      <Button
+        onButtonClick={() => dispatch(filterTask(TASK_FILTER_INCOMPLETED))}
+      >
+        Incomplete
+      </Button>
+      <Button onButtonClick={() => dispatch(filterTask(TASK_FILTER_COMPLETED))}>
+        Complete
+      </Button>
```

Comment:
Could we resolve this by not repeating same things again ?
```suggestion
     const filterStates = [{ name: 'All', status: TASK_FILTER_ALL}.....]
     {filterStates.map((filterState) => (
        <Button onButtonClick={() => dispatch(filterTask(filterState.status))}>
        filterState.name
      </Button>
      ))}
      
      
```